<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1404</width>
    <height>824</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>12</y>
      <width>1380</width>
      <height>755</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <layout class="QGridLayout" name="gridLayout">
      <item row="7" column="0" colspan="2">
       <widget class="QLabel" name="fitIterLabel">
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>MCMC Thin</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="MCMCWalkerLineEdit">
        <property name="text">
         <string>10</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="MCMCStepsLineEdit">
        <property name="text">
         <string>100</string>
        </property>
       </widget>
      </item>
      <item row="11" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Parameter Chains</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0" colspan="2">
       <widget class="QProgressBar" name="progressBar">
        <property name="value">
         <number>24</number>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="MCMCThinLineEdit">
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="ParallelCoresLineEdit">
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>MCMC Walkers</string>
        </property>
       </widget>
      </item>
      <item row="19" column="0" colspan="2">
       <widget class="QPushButton" name="calcConfIntervPushButton">
        <property name="text">
         <string>Calculate Confidence Interval</string>
        </property>
       </widget>
      </item>
      <item row="12" column="0" colspan="2">
       <widget class="QTreeWidget" name="parameterTreeWidget">
        <property name="selectionMode">
         <enum>QAbstractItemView::ExtendedSelection</enum>
        </property>
        <column>
         <property name="text">
          <string>Parameters</string>
         </property>
        </column>
       </widget>
      </item>
      <item row="10" column="1">
       <widget class="QPushButton" name="startSamplingPushButton">
        <property name="text">
         <string>Start Sampling</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="2">
       <widget class="QCheckBox" name="reuseSamplerCheckBox">
        <property name="text">
         <string>Re-use Sampler</string>
        </property>
       </widget>
      </item>
      <item row="10" column="0">
       <widget class="QPushButton" name="stopSamplingPushButton">
        <property name="text">
         <string>Stop Sampling</string>
        </property>
       </widget>
      </item>
      <item row="18" column="1">
       <widget class="QDoubleSpinBox" name="percentileDoubleSpinBox">
        <property name="decimals">
         <number>3</number>
        </property>
        <property name="value">
         <double>84.135000000000005</double>
        </property>
       </widget>
      </item>
      <item row="18" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Percentile</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Parallel Cores</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>MCMC Steps</string>
        </property>
       </widget>
      </item>
      <item row="17" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>MCMC Burn</string>
        </property>
       </widget>
      </item>
      <item row="17" column="1">
       <widget class="QLineEdit" name="MCMCBurnLineEdit">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="14" column="1">
       <widget class="QPushButton" name="removeUserDefinedParamPushButton">
        <property name="text">
         <string>Remove Parameter</string>
        </property>
       </widget>
      </item>
      <item row="13" column="0">
       <widget class="QLabel" name="userDefinedParamChainLabel">
        <property name="text">
         <string>User-Defined Parameter Chains</string>
        </property>
       </widget>
      </item>
      <item row="14" column="0">
       <widget class="QPushButton" name="addUserDefinedParamPushButton">
        <property name="text">
         <string>Add Parameter</string>
        </property>
       </widget>
      </item>
      <item row="16" column="0" colspan="2">
       <widget class="QTreeWidget" name="userDefinedParamTreeWidget">
        <property name="selectionMode">
         <enum>QAbstractItemView::ContiguousSelection</enum>
        </property>
        <column>
         <property name="text">
          <string>Parameters</string>
         </property>
        </column>
       </widget>
      </item>
      <item row="15" column="0">
       <widget class="QPushButton" name="saveUserDefinedParamPushButton">
        <property name="text">
         <string>Save Parameters</string>
        </property>
       </widget>
      </item>
      <item row="15" column="1">
       <widget class="QPushButton" name="loadUserDefinedParamPushButton">
        <property name="text">
         <string>Load Parameters</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="chainTab">
      <attribute name="title">
       <string>Parameter Chains</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="MplWidget" name="chainMPLWidget" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="userDefinedChainTab">
      <attribute name="title">
       <string>User-Defined Chains</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="MplWidget" name="userDefinedChainMPLWidget" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="correlationTab">
      <attribute name="title">
       <string>Parameter Correlations</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="MplWidget" name="correlationMPLWidget" native="true"/>
       </item>
       <item>
        <widget class="QLabel" name="label_8">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>Correlation time</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="corrTimeTextEdit">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>200</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="acceptFracTab">
      <attribute name="title">
       <string>Acceptance Fraction</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="MplWidget" name="acceptFracMPLWidget" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="cornerPlotTab">
      <attribute name="title">
       <string>Corner Plots</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="MplWidget" name="cornerPlotMPLWidget" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="confIntervalTab">
      <attribute name="title">
       <string>Confidence Interval</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0" colspan="2">
          <widget class="QTextEdit" name="confIntervalTextEdit"/>
         </item>
         <item row="1" column="1">
          <widget class="QPushButton" name="saveConfIntervalPushButton">
           <property name="text">
            <string>Save</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>468</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1404</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_MCMC_run_file"/>
    <addaction name="actionSave_MCMC_run_file"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen_MCMC_run_file">
   <property name="text">
    <string>Open MCMC run file</string>
   </property>
  </action>
  <action name="actionSave_MCMC_run_file">
   <property name="text">
    <string>Save MCMC run file</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MplWidget</class>
   <extends>QWidget</extends>
   <header>mplWidget</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
